tests/manifests/test_tabular.py

```diff
@@ -226,6 +226,46 @@ def test_time_type(tmp_path, rc):
     ''')
 
 
+def test_property_unique_add(tmp_path, rc):
+    check(tmp_path, rc, '''
+    d | r | b | m | property            | type
+    example                             |
+                                        |
+      |   |   | City                    |
+      |   |   |   | prop_with_unique    | string unique
+      |   |   |   | prop_not_unique     | string
+    ''')
+
+def test_property_unique_add_wrong_type(tmp_path, rc):
+    with pytest.raises(InvalidManifestFile) as e:
+        check(tmp_path, rc, '''
+        d | r | b | m | property | type
+        datasets/gov/example     |
+          |   |   | City         |
+          |   |   |   | value    | string unikue
+        ''')
+    assert e.value.context['error'] == (
+        "Unknown 'string unikue' type of 'value' property in 'datasets/gov/example/City' model."
+    )
+
+def test_property_with_ref_unique(tmp_path, rc):
+    check(tmp_path, rc, '''
+    d | r | b | m | property | type               | ref     | uri
+    datasets/gov/example     |                    |         |
+                             | prefix             | locn    | http://www.w3.org/ns/locn#
+                             |                    | ogc     | http://www.opengis.net/rdf#
+                             |                    |         |
+      | data                 | postgresql         | default |
+                             |                    |         |
+      |   |   | Country      |                    | code    |
+      |   |   |   | code     | string             |         |
+      |   |   |   | name     | string             |         | locn:geographicName
+                             |                    |         |
+      |   |   | City         |                    | name    |
+      |   |   |   | name     | string unique      |         | locn:geographicName
+      |   |   |   | country  | ref unique         | Country |
```

Comment:
```suggestion
      |   |   |   |          | unique             | name, country |
      |   |   |   | name     | string             |         | locn:geographicName
      |   |   |   | country  | ref                | Country |
```

Unique together, should be added on both `name` and `country` together, not separately.